submissions/description/broken_access_control/server_side_request_forgery_ssrf/dns_query_only/template.md

```diff
@@ -0,0 +1,27 @@
+# DNS-Only Server-Side Request Forgery
+
+## Overview of the Vulnerability
+
+Server-side request forgery (SSRF) is a vulnerability that exploits the trust relationship between a server and an application, or other backend systems. An attacker can take advantage of this trust to forge server-side traffic and make HTTP requests to internal or external domains. Such domains may include internal cloud server meta-data which consists of sensitive configuration information, internal HTTP database interfaces allowing database access, or using `file://` URI to read sensitive files. A SSRF vulnerability was found in this application which allows an attacker to observe DNS queries of the application.
```

Comment:
An SSRF Does not observe DNS queries, but requests them. I've updated the wording here.

> An SSRF vulnerability was found in this application which allows an attacker to request a DNS query from the application.